CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## [1.1.0] 9/22/22
 * Did cleanup after Python script additions. Minor formatting.
+* Removed flutter as a dependency allowing the library to be used in dart only projects.
```

Comment:
I think i'll modify this into a brand new note since i'll probably make a new release for this change.